### DISCUSSION TITLE
Examen final

### DISCUSSION THREAD

**Discussion by andreslb9** [mmasias/23-24-prg2#121] [General]
Hola, el examen va a ser a ordenador no?

**Comment by mmasias** [comment]
🖥️,📝&🧠

**Comment by mmasias** [comment]
Me preguntaron por email esto:

_Escribía para preguntarte por el examen de mañana, ya que somos muchos los que desconocemos su formato (papel y/o ordenador) y esta cuestión quedó en el aire en las últimas sesiones. También sería interesante conocer, en el caso de tener alguna parte escrita, si será similar al parcial de la asignatura o serán preguntas más teóricas. Entendemos que la parte más importante del examen será como los últimos retos realizados, un código implementando todo lo aprendido durante el curso.
Muchas gracias por tu atención. 
Un saludo,_

Contesté esto:

_Mañana, desarrollo exquisito, para lo cual hace falta todo eso: papel, ordenador, teoría y práctica. De todos modos, no será nada más (y por supuesto, nada menos) que lo que hemos ido viendo todo el cuatrimestre.
¡Yo creo que va a estar chulo!  :)_
